packages/hemera-nsq/README.md

```diff
@@ -30,29 +30,30 @@ const hemera = new Hemera(nats, {
   logLevel: 'info'
 })
 
-const options = {
-  reader: {
-    lookupdHTTPAddresses: 'http://127.0.0.1:4161'
+hemera.use(HemeraJoi)
+hemera.use(HemeraNsq, {
+  nsqReader: {
+    lookupdHTTPAddresses: [
+      'http://localhost:4161'
+    ]
   },
-  writer: {
-    url: '127.0.0.1',
-    port: 4150
+  nsqWriter: {
+    nsqdHost: 'localhost',
```

Comment:
The keys are already namespaced by `nsqWriter`, host and port would be better.

Reply:
Ty for review. I wanted to keep the same API as 'nsqjs' module (like 'nsqdTCPAddresses', 'lookupdHTTPAddresses'), but i can rewrite it)